### ISSUE TITLE
Cleaning House

### ISSUE THREAD

**Issue by nsoblath** [project8/katydid#45]
I propose to remove the following directories / files:
* Source;Data/Evaluation -- module is not used anymore and is based on pre-seeing-electrons assumptions.
* Source/EventAnalysis/KTCandidateCleaner & KTCandidateTrackFinder -- never implemented
* Source/EventAnalysis/KTMultiSliceClustering -- Old clustering algorithm; will never be used again
* Source/Simulation -- module is obsolete in light of the development of locust_mc
* Source/IO/KTBasicASCIIWriter & TypeWriterTS -- never fully implemented and not used
* Source/IO/KTJSONWriter & KTJSONMaker & TypeWriters -- no longer used
* Source/IO/KTMultiSliceROOTWriter & TypeWriters -- unnecessarily complicated way of doing this
* Source/IO/KTOfficialCandidatesWriter -- no longer used

If there are any objections, please let me know ASAP.

**Comment by laroque** [comment]
w.r.t. "Data/Evaluation" is that the one that was used to compare old locust_mc "truth" files against katydid candidates? While not in the near term plan, I can imagine wanting to do that again at some future time when locust is outputting egg files (maybe if we were simulating antenna response and trying to estimate effective volumes or some such).

**Comment by nsoblath** [comment]
I agree that we'll want to pursue this goal in the future.  However, I think it'll be more productive to start over, rather than try to reengineer this particular implementation, which was based on our pre-data assumptions about the track characteristics.

**Comment by nsoblath** [comment]
More things to do:
- [ ] Fix KTEggProcessor's documentation
- [ ] Fix the example configuration files; they're quite out-of-date
- [ ] Upgrade the processor summary signals/slots to KTDataPtr
- [ ] Turn on the Matlab egg reader by default
